Problem3/p3-1/driver/p3_1_arith.c

```diff
@@ -0,0 +1,17 @@
+/***************************** Include Files *******************************/
+#include "p3_1_arith.h"
+
+/************************** Function Definitions ***************************/
+u32 arith(UINTPTR baseAddr, u32 opd1, u32 opd2, u32 arith_mode)
+{
+    u32 result;
+
+	u32 opd1_mask = opd1 & 0xff; //mask to 8 bits
+	u32 opd2_mask = opd2 & 0xff; //mask to 8 bits
+	u32 arith_mode_mask = arith_mode & 0x11; //mask to 2 bits

```

Comment:
Here mask should be 0x3, or say, 0b11, rather than 0x11

Reply:
I modify mt code for this problem,
and the result looks correct now
![image](https://user-images.githubusercontent.com/99808959/165035169-86596325-ab69-4d30-9d27-deb6b8c68f34.png)